templates/designToDptCreation.jinja

```diff
@@ -29,11 +29,11 @@ bool {{functionPrefix}}createDpt{{cls.get('name')}}()
   
   {% for sv in cls.sourcevariable %}
     {% if sv.array|length>0 %}
-      {{debug("WARNING: Skipping array source-variable. There is no support for arrays in Cacophony yet. ",
+      {{debug("WARNING: Skipping array source-variable. There is no support for source variable arrays in Cacophony yet.",
          "You can request it from Piotr.")}}
     {% else %} {# scalar #}
       dynAppend(xxdepes, makeDynString("", "{{sv.get('name')}}"));
-      dynAppend(xxdepei, makeDynInt(0, {{mapper(sv.get('dataType'), class_name, sv.get('name'))}}));
+      dynAppend(xxdepei, makeDynInt(0, {{mapper(sv.get('dataType'), class_name, sv.get('name'), False)}}));
```

Comment:
I guess the "False" shouldn't be here since this is the scalar case of source variables and the function the quasar_data_type_to_dpt_type_constant(...) has the argument "noarray" set to "True" as default value.

Reply:
Yes if we remove it as norarray are True by default, array will not be treated. If we keep code as it is now this False is needed.
But we can change the quasar_data_type_to_dpt_type_constant code to remove the noarray argument

Reply:
Are you sure? I tottaly agree with the use of "False" in the line 23 that corresponds to array cache variables,
but this line corresponds to the scalar case of source variables.

Reply:
Yes, you have reason, this line concern source variable, where I don't implement array, so, the False here is wrong, sorry for the confusion